### DISCUSSION TITLE
Cannot upload proof of my academic status for a student discount

### DISCUSSION THREAD

**Discussion by ReOp14** [community/community#53814] [GitHub Education] [answered]
Hello, I'm getting extremely irritated with the AI refusing my student documents.
It's keeps giving the errors:

"You are unlikely to be verified until you have completed your with your full name as it appears in your academic affiliation document plus a short bio."
I really did not want to reveal my personal name on my profile but I did, and it still says the same error.

"Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy."
I tried this but the system would glitch out and give me no response and refresh the page. I uploaded my image that I took from my phone instead manually and it gives this error. A human would clearly see that its a printed document.

I also tried taking a picture and screenshot of the following items:
My School ID (it doesn't have a date on it)
My class schedule (Shows the current date and School and Classes)
A recent award letter (Shows my full name, a recent date and full school name)

Can I please have a human look at the documents? It really shouldn't be this difficult. I have attempted this like 9+ times by now. My school email should be enough to have confirmed I'm a current student.

**Comment by samus-aran** [comment, marked as answer]
Hi @ReOp14 !

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants.

The application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Hope this helps! Let us know how you get along 👍

**Comment by ReOp14** [comment]
Looks like the system doesn't bug out if doing it directly on the phone. Thank you.
